internal/fs/wrappers/monitoring.go

```diff
@@ -18,22 +18,33 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"log"
 	"syscall"
+	"time"
 
+	"github.com/googlecloudplatform/gcsfuse/internal/logger"
 	"github.com/googlecloudplatform/gcsfuse/internal/monitor/tags"
 	"github.com/jacobsa/fuse/fuseops"
 	"github.com/jacobsa/fuse/fuseutil"
+	"go.opencensus.io/plugin/ochttp"
 	"go.opencensus.io/stats"
 	"go.opencensus.io/stats/view"
 	"go.opencensus.io/tag"
 )
 
 var (
-	opCount = stats.Int64("fs/ops_count", "The number of ops processed by the file system.", stats.UnitDimensionless)
+	errorLogger       *log.Logger
```

Comment:
just to keep similar things together can we move errorLogger out of this grouping and keep it seperate